bindings/protocols/coap/index.html

```diff
@@ -54,86 +54,280 @@ <h2>Introduction</h2>
+            </p>
+            <pre id="example-observing-resources" class="example" title="Observing Resources">
+{
+    "@context": "https://www.w3.org/2022/wot/td/v1.1",
+    ...
+    "properties": {
+        "status": {
+            "type": "string",
+            "readOnly": true,
+            "forms": [{
+                "cov:method": "GET",
+                "href": "coap://[2001:DB8::1]/status",
+                "contentType": "text/plain;charset=utf-8",
+                "subprotocol": "cov:observe",
+                "op": ["observeproperty"]
+            }]
+        }
+    }
+}
+</pre>
+        </section>
+        <section>
+            <h3>Block-wise Transfers</h3>
+            <p>
+                CoAP supports block-wise transfers to allow large resource representations to be transferred between clients and servers.
+                This feature enables clients and servers to request or provide resource representations in smaller blocks, which can be useful when constrained network conditions make it undesirable to transfer large amounts of data at once.
+            </p>
+            <p>
+                At the time of writing, CoAP supports two mechanisms for block-wise transfers:
+                Block-Wise Transfers [[RFC7959]] and Block-Wise Transfer Options Supporting Robust Transmission [[RFC9177]].
+                In a form, support for these can be indicated by a <code>cov:blockwise</code> or <code>cov:quickblockwise</code> member, respectively.
+            </p>
+            <pre id="example-blockwise" class="example" title="Block-wise Transfer">
+{
+    "@context": "https://www.w3.org/2022/wot/td/v1.1",
+    ...
+    "properties": {
+        "status": {
+            "type": "string",
+            "readOnly": true,
+            "forms": [{
+                "href": "coap://[2001:DB8::1]/status",
+                "contentType": "text/plain;charset=utf-8",
+                "cov:blockwise": { }
+            }]
+        }
+    }
+}
+</pre>
+            <p>
+                Additionally, a <code>cov:blockwise</code> or <code>cov:quickblockwise</code> member may indicate relevant parameters, such as the largest block size that may be used in a <code>Block2</code> Option.
+            </p>
+            <pre id="example-blockwise-parameters" class="example" title="Block-wise Transfer with Parameters">
+{
+    "@context": "https://www.w3.org/2022/wot/td/v1.1",
+    ...
+    "properties": {
+        "status": {
+            "type": "string",
+            "readOnly": true,
+            "forms": [{
+                "href": "coap://[2001:DB8::1]/status",
+                "contentType": "text/plain;charset=utf-8",
+                "cov:blockwise": {
+                    "cov:block2SZX": 64
+                }
+            }]
+        }
+    }
+}
+</pre>
+        </section>
+        <section>
+            <h3>Conditional Requests</h3>
```

Comment:
Some examples of this in a TD would be nice. I understand this mechanism but do not see a direct use case in WoT? Can it be used for doing something like "turn off the ventilator if the temperature property is lower than 25"? This would allow the Consumer to not make a read request beforehand.

Reply:
Remember that this section describes the features of CoAP first and how they can be used in a TD second. I don't think we should leave off Conditional Requests from the document, because then readers might be wondering about those.

The issue is that you wouldn't drive the use of Conditional Requests directly from a Thing Description. You cannot use it "for doing something like 'turn off the ventilator if the temperature property is lower than 25'".

So we describe the feature, but provide no vocabulary. I've clarified the text to say so.